### DISCUSSION TITLE
yandex添加后能看到列表，但文件无法访问

### DISCUSSION THREAD

**Discussion by soundghost** [AlistGo/alist#2573] [Q&A]
比如网盘里的txt文件，直接点击出现：
```Failed to fetch https://www.exsample.com/p/Yandex/Windows10Tools/%E4%BF%AE%E5%A4%8Dsamba.txt: AxiosError: Network Error```
压缩包打开能看到大小，点击下载出现：
```Server Closed Connection Suddenly.```

请问是什么原因？

**Comment by woshichenghaibo** [comment]
我的问题一样

**Comment by anwen-anyi** [comment]
`Network Error` 不出意外的话出意外了 `AList服务端`应该需要代理才行

**Comment by woshichenghaibo** [comment]
我的服务器就在俄罗斯
________________________________
发件人: 安稳 ***@***.***>
发送时间: 2023年6月14日 18:53
收件人: alist-org/alist ***@***.***>
抄送: woshichenghaibo ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/alist] yandex添加后能看到列表，但文件无法访问 (Discussion #2573)


Network Error 不出意外的话出意外了 AList服务端应该需要代理才行

―
